.ci/install_kata.sh

```diff
@@ -53,12 +54,18 @@ case "${KATA_HYPERVISOR}" in
 		"${cidir}/install_cloud_hypervisor.sh"
 		echo "Installing experimental_qemu to install virtiofsd"
 		install_qemu
+		if [[ "${arch}" == "x86_64" ]]; then
```

Comment:
Nits:

- You could use a single set of square brackets throughout as I think the only time you need them is for regex matching (`[[ $thing =~ pattern ]]`).
- You could simplify these tests to remove the `if`:
  ```bash
  [ "${arch}" = "x86_64" ] && "${cidir}/install_virtiofsd.sh"
  ```

Reply:
Updated, thanks!